json_serializable/lib/src/constants.dart

```diff
@@ -6,7 +6,6 @@
 const closureArg = 'e';
 
 const generatedLocalVarName = 'val';
```

Comment:
+1 on renaming to `'value'` while we're changing the generated output.

Reply:
done